modules/minimum-required-permissions-ipi-gcp.adoc

```diff
@@ -28,10 +28,17 @@ If your organization’s security policies require a more restrictive set of per
 * `compute.forwardingRules.get`
 * `compute.forwardingRules.list`
 * `compute.forwardingRules.setLabels`
+* `compute.globalAddresses.create`
+* `compute.globalAddresses.get`
+* `compute.globalAddresses.setLabels`
```

Comment:
During [my testing](https://issues.redhat.com/browse/OCPBUGS-34129?focusedId=24938938&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24938938), the permission is not required.

Reply:
@jianli-wei Did you test with a CAPG install? I found that these permissions were required as the resources created by CAPG are global (not regional).

Reply:
@barbacbd Sorry for the late reply, yes, I did test with CAPG/CAPI install, the permission `compute.globalAddresses.setLabels` is not needed. Thanks!